src/modules/fancyzones/lib/ZoneSet.cpp

```diff
@@ -366,45 +381,91 @@ ZoneSet::MoveWindowIntoZoneByDirectionAndIndex(HWND window, HWND windowZone, DWO
 }
 
 IFACEMETHODIMP_(bool)
-ZoneSet::MoveWindowIntoZoneByDirectionAndPosition(HWND window, HWND windowZone, DWORD vkCode, bool cycle) noexcept
+ZoneSet::MoveWindowIntoZoneByDirectionAndPosition(HWND window, HWND windowZone, DWORD vkCode, bool cycle, bool selectManyZones) noexcept
```

Comment:
function name is misleading now as it not necessary moves the window. It would be nice to make 2 separate functions out of this one